app/src/organisms/ErrorRecoveryFlows/constants.ts

```diff
@@ -333,6 +357,15 @@ export const RECOVERY_MAP_METADATA: RecoveryRouteStepMetadata = {
   [ROBOT_DOOR_OPEN.ROUTE]: {
     [ROBOT_DOOR_OPEN.STEPS.DOOR_OPEN]: { allowDoorOpen: false },
   },
+  [HOME_AND_RETRY.ROUTE]: {
+    [HOME_AND_RETRY.STEPS.PREPARE_DECK_FOR_HOME]: { allowDoorOpen: true },
+    [HOME_AND_RETRY.STEPS.REMOVE_TIPS_FROM_PIPETTE]: { allowDoorOpen: true },
```

Comment:
```suggestion
    [HOME_AND_RETRY.STEPS.REMOVE_TIPS_FROM_PIPETTE]: { allowDoorOpen: false },
```
I think we probably want to keep the door closed during drop tip wizard. I personally haven't tested it, but because DT Wiz is essentially its own API, I'm not certain it will play well with the door open and popping the door open modal. 

Might be worth a discussion, but for safety reasons as well as copy considerations, maybe want to default to `allowDoorOpen: false` unless the step specifically says the door is open?

Reply:
That makes sense. I think all the other ones do actually want to allow an open door, though. We do special handling for the "click to home" button.